### DISCUSSION TITLE
How to monitor application hosted on cloud and sky walking is on premise

### DISCUSSION THREAD

**Discussion by ccfahe** [apache/skywalking#9410] [Q&A]
My microservice application is hosted on Cloud Azure and my skywalking is on a physical server. How can I monitor the application and the steps used for connection?IS it possible?

Do it work for all cloud(AWS,Azure,GCP)

**Comment by wu-sheng** [comment]
I don't what makes you feel cloud is special.

**Reply by ccfahe** [reply]
Our applications are hosted over servers and well as cloud so we are looking for a tool who can monitor both env.

**Reply by wu-sheng** [reply]
Observability and monitoring are tech stack oriented, what your ask seems not relative.
